pkg/pgmodel/querier/series_set.go

```diff
@@ -81,10 +97,21 @@ func (p *pgxSeriesSet) At() storage.Series {
 	// this should pretty much always be non-empty due to __name__, but it
 	// costs little to check here
 	if len(row.labelIds) != 0 {
```

Comment:
I think it will be better to have this as `if len(row.labelIds) == 0` and simply return ps, this will decrease the nesting and look cleaner.